packages/@webex/plugin-authorization-browser-first-party/test/unit/spec/authorization.js

```diff
@@ -223,7 +223,7 @@ describe('plugin-authorization-browser-first-party', () => {
           assert.calledOnce(requestAuthorizationCodeGrantStub);
           assert.calledWith(requestAuthorizationCodeGrantStub, {code, codeVerifier: undefined});
           assert.calledOnce(collectPreauthCatalogStub);
-          assert.calledWith(collectPreauthCatalogStub, undefined);
+          assert.calledWith(collectPreauthCatalogStub, {orgId: 'theOrgId'});
         });
 
```

Comment:
maybe add a test for unexpected auth code format / undefined?  Or will that never happen?

Reply:
I've updated the code to handle the no orgId case. We'll use emailhash, then orgId and then no params in that order. This can be the case for certain account types